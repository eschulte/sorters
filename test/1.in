1 4 56 2 43 8 76 12 43 7