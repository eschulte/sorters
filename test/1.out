1 2 4 7 8 12 43 43 56 76
